### PR TITLE
nitunit: Fix documentation to reflect actual test discovery algorithm

### PR DESCRIPTION
**Pull request by jcbrinfo**
Signed-off-by: Jean-Christophe Beaupré <jcbrinfo@users.noreply.github.com>

### CODE REVIEW

**Comment by privat** [comment]
ping @Morriar

**Comment by privat** [comment]
Seems good

Note that I'm still working on nitunit (e.g #2167) so I will have to solve merge conflicts. (should be easy here)

File: share/man/nitunit.md [hunk 1]
```diff
@@ -129,18 +129,25 @@ This flag can be used by libraries and program to prevent (or limit) the executi
...
 ~~~~
 # test suite for module `foo`
-module test_foo
+module test_foo is test_suite
+
+import test_suite
 import foo # can be intrude to test private things
+
 class TestFoo
+	super TestSuite
```
Comment (Morriar) [inline share/man/nitunit.md]: Indentation looks weird compared to the rest of the source code

**Comment by Morriar** [comment]
+1

**Pull request merged**